Plugins/Gource/GourceStart.cs

```diff
@@ -155,7 +155,7 @@ private void WorkingDirBrowseClick(object sender, EventArgs e)
 
         private void linkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
         {
-            Process.Start(@"https://github.com/acaudwell/Gource/");
+            OsShellUtil.OpenUrlInDefaultBrowser(@"https://github.com/acaudwell/Gource/");
```

Comment:
Can `using System.Diagnostics;` be removed?

Reply:
Maybe yes. I will try to have a look before being not available during a week...

Reply:
In fact, no, that's not possible due to a Process.Start to launch gource executable:
https://github.com/gitextensions/gitextensions/blob/2be645a4d66560dc6d7adef45520d476ba7dd80c/Plugins/Gource/GourceStart.cs#L46